source/codegen/templates/service_helpers.mako

```diff
@@ -19,7 +20,11 @@ ${initialize_input_params(function_name, parameters)}
       };
       uint32_t session_id = 0;
       const std::string& session_name = request->session_name();
+      %if 'custom_close' in function_data:
+      auto cleanup_lambda = [&] (uint32_t id) { library_->${function_data['custom_close']}; };
+      %else:
       auto cleanup_lambda = [&] (uint32_t id) { library_->${config['close_function']}(id); };
+      %endif
```

Comment:
This change is also made in one of the PR which was targetted to `features/ni-dmm`.
We will take care of it while merging it in the ni-repo.